style.css

```diff
@@ -109,7 +110,7 @@ header {
   justify-content: center;
   align-items: center;
   align-content: center;
-  margin: 1rem auto 0 auto;
+  margin: 0 auto;
 }
 
```

Comment:
# 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 
- **Kindly update your ``README`` file.**
- **Fill the missing information:**
<img width="1440" alt="Screen Shot 2021-11-13 at 3 21 31" src="https://user-images.githubusercontent.com/22258274/141613542-d4dc7bdd-ccf3-4f44-913b-afd1286afa26.png">